plugins/a11y-keyboard/msd-a11y-preferences-dialog.c

```diff
@@ -49,6 +49,7 @@
 #define KEY_BOUNCE_KEYS_ENABLED      "bouncekeys-enable"
 #define KEY_SLOW_KEYS_ENABLED        "slowkeys-enable"
 #define KEY_MOUSE_KEYS_ENABLED       "mousekeys-enable"
+#define KEY_CAPSLOCK_BEEP_ENABLED    "capslock-beep-enable"
```

Comment:
This macro has been defined in another file.

Reply:
Yes, but I'm quite unsure where we should share those.  Actually we could share quite a few of them instead of hard-coding several strings, but that's IMO a separate refactoring to do (or at least it should be done for other setting name strings in the manager -- e.g. the string `"stickykeys-enable"` appears 3 times, that would warrant a macro, and it could be shared somehow).

Reply:
Yes, apparently we have a problem with the constants, which are defined in .c files and some of them are not used, so you're right and this can be fixed in another request by moving all constant to one .h file like at-spi2-core does:
```
atspi/atspi-constants.h 
```